src/main/webapp/WEB-INF/tags/instructor/home/coursePanel.tag

```diff
@@ -5,8 +5,8 @@
 <%@ attribute name="courseTable" type="teammates.ui.template.CourseTable" required="true" %>
 <%@ attribute name="index" required="true" %>
 <c:set var="isNotLoaded" value="${empty courseTable.buttons}" />
-<div class="panel panel-primary" id="course-${index}"<c:if test="${isNotLoaded}"> style="cursor: pointer;"</c:if>>
-    <div class="panel-heading">
+<div class="panel panel-primary" id="course-${index}">
+    <div class="panel-heading ajax_auto" <c:if test="${isNotLoaded}"> style="cursor: pointer;"</c:if>>
```

Comment:
binding was done every where on panel-heading, so just to match the format I did that. I will move this up.